ajax/libs/jspanel/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jspanel",
+  "version": "2.6.3",
+  "filename": "jquery.jspanel.min.js",
+  "description": "A jQuery Plugin to create highly configurable multifunctional floating panels",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Flyer53/jsPanel.git"
+  },
+  "keywords": [
+    "jquery-plugin",
+    "jsPanel",
+    "panel",
+    "ui",
+    "layout",
+    "popup",
+    "widget",
+    "window",
+    "modal",
+    "tooltip",
+    "hint",
+    "bootstrap"
+  ],
+  "author": "Stefan Straesser <info@jspanel.de> (http://jspanel.de/)",
+  "homepage": "http://jspanel.de",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Flyer53/jsPanel.git",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+          "*+(jspanel|jsPanel)!(*-[0-9])*",
```

Comment:
I think it's fine. both of them can get the files we want.
As this pattern `+(*jspanel|jsPanel|jsglyph)!(*-[0-9])*` just `*jspane` can get the `*jspane` files, but `jsPanel` just get the `jsPanel*` files, if some filename `*jsPanel`, then we can't get them.